### PR TITLE
Updates look and feel of GCS bucket content browser

### PR DESCRIPTION
**Pull request by etanshaul**
Updates the table look and feel containing the GCS bucket contents.

Before:

image

After:

image

image

### CODE REVIEW

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobNameCellRenderer.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import java.awt.Component;
+import javax.swing.JTable;
+import javax.swing.table.DefaultTableCellRenderer;
+import javax.swing.table.TableCellRenderer;
+
+/** Custom {@link TableCellRenderer} for rendering blob name cells. */
+final class GcsBlobNameCellRenderer extends DefaultTableCellRenderer {
+
+  GcsBlobNameCellRenderer() {
+    setOpaque(true);
```
Comment (nkibler) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobNameCellRenderer.java]: Should this be done in `getTableCellRendererComponent()`? Wondering if it's necessary to be done here, considering the other setters are done below.

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanel.java [hunk 2]
```diff
@@ -40,7 +45,9 @@
   private JTable bucketContentTable;
   private GcsBlobTableModel tableModel;
 
-  GcsBucketContentEditorPanel(@NotNull Bucket bucket) {
+  private static final Color MEDIUM_GRAY = new Color(96, 96, 96);
+
+      GcsBucketContentEditorPanel(@NotNull Bucket bucket) {
```
Comment (nkibler) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanel.java]: nit: Looks like the indentation here got messed up

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobNameCellRenderer.java (hunk 1 above)
Comment (etanshaul) [reply google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobNameCellRenderer.java]: it looks like this isn't needed at all any more. Not 100% why, but I'll remove it.

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanel.java (hunk 2 above)
Comment (etanshaul) [reply google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanel.java]: fixed

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobNameCellRenderer.java (hunk 1 above)
Comment (etanshaul) [reply google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/gcs/GcsBlobNameCellRenderer.java]: done

**Comment by etanshaul** [comment]
@nkibler comments addressed

**Pull request merged**